tools/atip/atip/web/steps.py

```diff
@@ -209,6 +209,35 @@ def click_element_by_link(context, text):
     return False
 
 
+@step(u'repeat to download resources from link "{linktext}" for {timeout:d} seconds')
```

Comment:
Please also update the README file which is located at https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/tools/atip/README.md